### PR TITLE
RELATED: FET-249 Use docker image with node 12 LTS

### PR DESCRIPTION
**Pull request by xMort**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by xMort** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2464/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2844/ : FAILURE in 1m 26s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6632/ : FAILURE in 2m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2463/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6635/ : FAILURE in 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2469/ : SUCCESS in 3s

File: src/components/visualizations/headline/utils/test/calculateXirr.spec.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ describe("calculateXirr", () => {
         ];
 
         const actual = calculateXirr(xirrInput);
-        const expected = 0.37336253350955556;
+        const expected = 0.3733625335095555;
```
Comment (xMort) [inline src/components/visualizations/headline/utils/test/calculateXirr.spec.ts]: Arithmetics were changed somewhere between four major versions of node.js we are upgrading to. Confirmed with @no23reason that it's OK in this case.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6637/ : FAILURE in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2471/ : SUCCESS in 3s

**Comment by xMort** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6638/ : FAILURE in 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2472/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6644/ : FAILURE in 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2479/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6645/ : FAILURE in 2m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2480/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6646/ : FAILURE in 2m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2481/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6647/ : SUCCESS in 9m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2482/ : SUCCESS in 3s

**Comment by xMort** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3462/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2483/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2850/ : SUCCESS in 19m 26s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3463/ : SUCCESS in 19m 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6648/ : SUCCESS in 6m 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1606/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2309/ : SUCCESS in 20m 33s

**Pull request merged**